pymusiccast/helpers.py

```diff
@@ -4,16 +4,18 @@
 import time
 import logging
 import requests
+import re
 _LOGGER = logging.getLogger(__name__)
 
 
 def request(url, *args, **kwargs):
     """Do the HTTP Request and return data"""
-    method = kwargs.get('method', 'GET')
+    method = kwargs.pop('method', 'GET')
     timeout = kwargs.pop('timeout', 10)  # hass default timeout
     req = requests.request(method, url, *args, timeout=timeout, **kwargs)
     data = req.json()
-    _LOGGER.debug(json.dumps(data))
+    ip = re.findall(r'[0-9]+(?:\.[0-9]+){3}', url)
```

Comment:
I think using a regex here is a bit oversized. How about using urlparse() to determine the netloc?

Reply:
Using urlparse now. Thanks for the tip